src/driver_client/dclient.rs

```diff
@@ -132,6 +131,19 @@ impl DriverClient {
         Ok(())
     }
 
+    pub fn reset_sensor_data(&self) -> Result<()> {
+        let ctrl_reg = self.ctrl_read_u32(
+            self.cfg.ctrl_cms_baseaddr + 0x028000,
+            CMS_ADDR::ADDR_HIF2CPU_CMS_CONTROL_REG,
+        );
+        self.ctrl_write_u32(
+            self.cfg.ctrl_cms_baseaddr + 0x028000,
+            CMS_ADDR::ADDR_HIF2CPU_CMS_CONTROL_REG,
+            ctrl_reg.unwrap() | 1,
+        )?;
+        Ok(())
+    }
```

Comment:
What sensors are we reseting here?

Should we move the `0x028000` to a const for readability?

Reply:
I moved these to const in some other prs, i suggest we do that

Reply:
you are right, I update this constants regarding Xilinix CMS product guide